exporters/ostream/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ include_directories(include)
 add_library(opentelemetry_exporter_ostream_metrics src/metrics_exporter.cc)
 add_library(opentelemetry_exporter_ostream_span src/span_exporter.cc)
 
+if(BUILD_TESTING)
 add_executable(ostream_metrics_test test/ostream_metrics_test.cc)
 add_executable(ostream_span_test test/ostream_span_test.cc)
```

Comment:
Thanks for this fix. Let's please indent this block, so it's easier to read.

Reply:
Was torn back and forth if i should ident it or not, but overwrote the commit to keep a cleaner history.